index.html

```diff
@@ -0,0 +1,290 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,400&family=Poppins:wght@300;400;700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <script src="https://kit.fontawesome.com/38b6d4ab77.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <!-- navbar section -->
+        <a href="#" class="logo-class">My Logo</a>
+        <nav>
+            <ul class="navbar-item">
+                <li class="navbar-links-part"><a href="#portfolio-link">Portfolio</a></li>
+                <li class="navbar-links-part"><a href="#about-link">About</a></li>
+                <li class="navbar-links-part"><a href="#contact-link">Contact</a></li>
+            </ul>
+        </nav>
+        
+            <ul id="navbar-item-mobile">
+                <li><a href="#" class="navbar-links-partt nav-item" id="close-menu">X</a></li>
+                <li class="navbar-links-part nav-mobile"><a href="#portfolio-link" class="nav-a">Portfolio</a></li>
+                <li class="navbar-links-part nav-mobile"><a href="#about-link" class="nav-a">About</a></li>
+                <li class="navbar-links-part nav-mobile"><a href="#contact-link" class="nav-a">Contact</a></li>
+            </ul>
+            <a href="#" class="humbargar" id="open-menu"><i class="fa-solid fa-bars"></i></a>
```

Comment:
I have realized that the following sections of your mobile menu do not intuitively match the provided Figma design. Below I will list them one by one.

- [ ] When the popup is clicked, the "light blue" background should have a form of a blur. In your case, that blur seems to be missing. This can be easily implemented by making use of the CSS backdrop filter property. Kindly have a look at [this resource](https://www.w3schools.com/cssref/css3_pr_backdrop-filter.php) I also use each time I need to implement this feature.
- [ ] Your menu options ie the Portfolio, about etc seem to not be having the correct font weights and sizes as the ones provided in the Figma design. Kindly update the font sizes used to match the ones in the Figma design.
- [ ] Adding to the above, the position of the menu options also seems to not be the same as the one as the one that is provided in the Figma design. KIndly update the margins/ padding accordingly so tat they get positioned intuitively on the same position as the one provided in the Figma design.

**Your design**

![Screenshot from 2023-06-08 19-54-43](https://github.com/sadaf-Daneshgar/First-Project-Microverse/assets/86318284/d650e665-a1fb-4064-a3b0-15519817c449)

**The Figma design:**

![Screenshot from 2023-06-08 19-53-41](https://github.com/sadaf-Daneshgar/First-Project-Microverse/assets/86318284/bba8ff3b-9177-466b-9832-d3150cf9967c)

**Note the difference?**

Reply:
Hello 
Kindly should I submit again when I add this changes and if yes where?